airlock_processor/BlobCreatedTrigger/function.json

```diff
@@ -0,0 +1,21 @@
+{
+  "scriptFile": "__init__.py",
+  "entryPoint": "main",
+  "bindings": [
+    {
+      "name": "msg",
+      "type": "serviceBusTrigger",
```

Comment:
if this is triggered by a service bus message why are we calling the function `BlobCreatedTrigger` ?

Reply:
ok, makes sense with the context, ta.